packages/components/src/components/Table/Table.stories.tsx

```diff
@@ -11,80 +12,100 @@ import {
   SortingState,
 } from "@tanstack/react-table";
 import map from "lodash/map";
-import { Box } from "../../primitives/Box";
 import { Table, THead, TBody, Tr, Th, Td } from "./index";
 
-export default {
-  component: Table,
+const table: Meta<typeof Table> = {
+  /* 👇 The title prop is optional.
+   * See https://storybook.js.org/docs/7.0/react/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
   title: "Components/Table",
-} as ComponentMeta<typeof Table>;
+  component: Table,
+};
 
-const Template: ComponentStory<typeof Table> = () => {
-  return (
-    <Table tableLayout="fixed">
+export default table;
+
+type Story = StoryObj<typeof Table>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/7.0/react/api/csf
+ * to learn how to use render functions.
+ */
```

Comment:
Same here, remove the comment. I created a Linear ticket to investigate storybook improvements.